lint-staged.config.js

```diff
@@ -6,9 +6,9 @@ const lintStaged = {
   // Lint and format TypeScript and JavaScript files
   '*.(ts|tsx|js|jsx|mjs)': (filenames) => [
     // some extra handling to use next's eslint https://github.com/vercel/next.js/issues/27997#issuecomment-900554790
-    `yarn lint --fix --file ${filenames
-      .map((file) => file.split(process.cwd())[1])
-      .join(' --file ')}`,
+    `yarn lint:files --fix ${filenames
+      .map((file) => file.split(`${process.cwd()}/`)[1])
+      .join(' ')}`,
```

Comment:
I added a new `lint:files` script so we could pass individual files to it for use in `lint-staged`. This is because we previously were using `next lint` for our linting, but that's been deprecated in favor of straight-up `eslint`.

Along the way, I also discovered that the previous version of this was passing `/src/absolute/paths.ts` instead of relative paths, which understandably caused `eslint` to get confused. Apparently the `next lint` command didn't really care about passing absolute paths and just assumed you meant to give it a relative one? 🤷 At any rate, I had to make it pass relative paths by adding that trailing `/` to the `file.split()`.

Reply:
🤔 That's not a bad idea. I only added that new script because I didn't even think about running _not_ a script in here. 😅 

If I had to think of an advantage, though, it'd be visibility + future changes. If we decide to modify the script to add some other CLI options / flags later. The `yarn eslint --fix ...` in this file is a little hidden unless you remember to go look for it. It's not a strong advantage, but if I had to think of one, that's all I've got. 😜